services/httpd/service.go

```diff
@@ -2,12 +2,28 @@ package httpd
 
 import (
 	"crypto/tls"
+	"expvar"
 	"fmt"
 	"log"
 	"net"
 	"net/http"
 	"os"
 	"strings"
+
+	"github.com/influxdb/influxdb"
+)
+
+// statistics gathered by the httpd package.
+const (
+	statRequest                   = "req"                 // Number of HTTP requests served
+	statCQRequest                 = "cq_req"              // Number of CQ-execute requests served
+	statQueryRequest              = "query_req"           // Number of query requests served
+	statWriteRequest              = "write_req"           // Number of write requests serverd
+	statPingRequest               = "ping_req"            // Number of ping requests served
+	statWriteRequestBytesReceived = "write_req_bytes"     // Sum of all bytes in write requests
```

Comment:
It would be nice to also know the total bytes written as well.  You could do this from the logging package and just look for when the route name is `write`.